Jenkinsfile_CNP

```diff
@@ -10,13 +13,51 @@ def component = "payment-processor"
 def channel = '#bsp-build-notices'
 GradleBuilder builder = new GradleBuilder(this, product)
 
+def secrets = [
+  'bulk-scan-${env}': [
+    secret('payments-queue-listen-connection-string', 'PAYMENTS_QUEUE_READ_CONNECTION_STRING')
+  ]
```

Comment:
do you need this?
inline 48 you get this again?

Reply:
As nitin suggested earlier, I'll remove reading the secret in line 48. so we'll need this